### ISSUE TITLE
starting up master branch of dat-desktop , nothing shows up

### ISSUE THREAD

**Issue by plankatron** [dat-land/dat-desktop#631]
[10126:0406/132754.968765:ERROR:CONSOLE(7746)] "Extension server error: Object not found: <top>", source: chrome-devtools://devtools/bundled/shell.js (7746)

**Comment by tpaixao** [comment]
same on Ubuntu Mate 19.

**Comment by aeriform-io** [comment]
Looking into this. On MacOS 10.14.5. Terminal reports:

```
Attempted to load extension "React Developer Tools" that has already been loaded.
2020-02-15 09:57:31.770 Electron[25405:3830135] *** WARNING: Textured window <AtomNSWindow: 0x7ff3d8e62400> is getting an implicitly transparent titlebar. This will break when linking against newer SDKs. Use NSWindow's -titlebarAppearsTransparent=YES instead.
dyld: lazy symbol binding failed: Symbol not found: __ZN2v816FunctionTemplate3NewEPNS_7IsolateEPFvRKNS_20FunctionCallbackInfoINS_5ValueEEEENS_5LocalIS4_EENSA_INS_9SignatureEEEiNS_19ConstructorBehaviorENS_14SideEffectTypeE
  Referenced from: /Users/aeriform/Documents/dat-desktop/node_modules/sodium-native/prebuilds/darwin-x64/electron.abi64.node
  Expected in: flat namespace

dyld: Symbol not found: __ZN2v816FunctionTemplate3NewEPNS_7IsolateEPFvRKNS_20FunctionCallbackInfoINS_5ValueEEEENS_5LocalIS4_EENSA_INS_9SignatureEEEiNS_19ConstructorBehaviorENS_14SideEffectTypeE
  Referenced from: /Users/aeriform/Documents/dat-desktop/node_modules/sodium-native/prebuilds/darwin-x64/electron.abi64.node
  Expected in: flat namespace

[25405:0215/095733.209172:ERROR:CONSOLE(7746)] "Extension server error: Object not found: <top>", source: chrome-devtools://devtools/bundled/shell.js (7746)
Hash: 05031ecfbb55473977bf
Version: webpack 4.41.6
```

Deps appear to be very far behind.

**Comment by aeriform-io** [comment]
I updated Electron to latest and got it working. It's giving deprecation and security warnings but no errors I can see. But nothing happens when clicking "Get Started".

<img width="917" alt="Screenshot 2020-02-15 10 43 40" src="https://user-images.githubusercontent.com/15712187/74586551-95a35780-4fe0-11ea-8332-eb7859b7999e.png">

**Comment by okdistribute** [comment]
This should be fixed on master, coming in  2.1 release!
